table/tables/tables.go

```diff
@@ -673,6 +661,27 @@ func (t *Table) IterRecords(ctx context.Context, startKey kv.Key, cols []*table.
 	return nil
 }
 
+func GetColDefaultValue(ctx context.Context, col *table.Column, defaultVals []types.Datum) (
```

Comment:
I think you can implement a GetAllColDefaultValues to return default values at one time. Passing a slice every time is a little strange.